terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -298,6 +321,146 @@ resource "kubernetes_cron_job" "workflow_manager" {
   }
 }
 
+resource "kubernetes_deployment" "intake_batch" {
+  metadata {
+    name      = "intake-batch-${var.ingestor}"
+    namespace = var.kubernetes_namespace
+  }
+
+  spec {
+    replicas = var.intake_batch_replica_count
+    selector {
+      match_labels = {
+        name = "intake-batch-${var.ingestor}"
+      }
+    }
+    template {
+      metadata {
+        labels = {
+          name = "intake-batch-${var.ingestor}"
+        }
+      }
+      spec {
+        service_account_name = module.account_mapping.kubernetes_account_name
+        container {
+          name  = "facile-container"
+          image = "${var.container_registry}/${var.facilitator_image}:${var.facilitator_version}"
+          args  = ["intake-batch-worker"]
+          resources {
+            # Batch intake is single-threaded, so we request an entire CPU for
+            # maximum throughput. We never expect to see batches larger than
+            # 3-400 MB, and batches are processed in memory, so request enough
+            # for whole ingestion batches plus a safety margin.
+            requests {
+              memory = "500Mi"
+              cpu    = "1.0"
+            }
+            limits {
```

Comment:
With having requests and limits so close to each other you limit the possibility for bin packing and better resource utilization. The limits are just there to basically tame possible run-away's and drive them into OOMs or starve them of hogging CPU. The requests are there to help a bit the bin packing if you know the lower end of resource requirements well.